policyengine_us/variables/gov/hhs/medicaid/eligibility/is_medicaid_immigration_status_eligible.py

```diff
@@ -17,20 +18,11 @@ def formula(person, period, parameters):
         immigration_status = person("immigration_status", period)
         immigration_status_str = immigration_status.decode_to_str()
 
-        # Check if immigration status is in the eligible list
+        # Check if immigration status is in the eligible list for federal Medicaid
+        # State-specific coverage for undocumented immigrants is now handled
+        # by separate state program variables (e.g., is_ca_state_medicaid_eligible)
```

Comment:
This change will break all of the other states outside of CA. We should find a way to transition California by itself without excluding all other states